File: Library.Core.Web/src/app/view-models/brochure/getBrochureView.ts
export class GetBrochureView {
  constructor(
    public brochures: BrochureGetBrochureViewItem[]
  ) { }
}

export class BrochureGetBrochureViewItem {
  constructor(
    public id: number = 0,
    public name?: string,
    public typeOfCover?: string,
    public numberOfPages: number = 0
  ) { }
}

File: Library.Core.ViewModelLayer/ViewModels/BrochureViewModels/GetBrochureView.cs
using System.Collections.Generic;

namespace Library.Core.ViewModelLayer.ViewModels.Brochure
{
    public class GetBrochureView
    {
        public ICollection<BrochureGetBrochureViewItem> Brochures { get; set; }

        public GetBrochureView()
        {
            Brochures = new List<BrochureGetBrochureViewItem>();
        }
    }

    public class BrochureGetBrochureViewItem
    {
        public int BrochureId { get; set; }
        public string Name { get; set; }
        public string TypeOfCover { get; set; }
        public int? NumberOfPages { get; set; }
    }
}
